docs/guidance/contracts-to-projects.md

```diff
@@ -104,28 +104,28 @@ Keep a copy each time the data changes, and if you see variations to:
 
 #### (6) Add project completion data
 
-When there is evidence that a project has reached completion, it is important to further update the **project level disclosure**. 
+When there is evidence that a project has reached completion, it is important to further update the **project level disclosure**.
 
-If the OCDS data includes implementation data, including transactions or final spending information, then it may be possible to compare the total sum of all contract spending against the original anticipated contract spend, and overall project budget. It may also be possible to compare final contract delivery dates with originally planned dates. This can be used to identify possible variations that are in need to explanation. 
+If the OCDS data includes implementation data, including transactions or final spending information, then it may be possible to compare the total sum of all contract spending against the original anticipated contract spend, and overall project budget. It may also be possible to compare final contract delivery dates with originally planned dates. This can be used to identify possible variations that are in need to explanation.
 
-In other cases, you may need to identify other data sources (such as treasury or public spending data) that you can draw upon to check whether a project spend was as anticipated or not. 
+In other cases, you may need to identify other data sources (such as treasury or public spending data) that you can draw upon to check whether a project spend was as anticipated or not.
 
 ### Tools and platform
 
-You can use OCDS data as part of a manual monitoring process, or you can integrate OCDS into a comprehensive transparency portal. 
+You can use OCDS data as part of a manual monitoring process, or you can integrate OCDS into a comprehensive transparency portal.
 
 Tools to help you with manual monitoring include:
 
 * [OCDS Kingfisher](https://github.com/open-contracting/kingfisher/) - a framework for regularly fetching, storing and querying OCDS data.
 * [OCDS Merge](https://github.com/open-contracting/ocds-merge) - a library to combine multiple releases of OCDS data into a summary (compiledRelease), and to identify changes over time (versionedReleased).
-* [OCDS Show](https://github.com/open-contracting/ocds-show) - a flexible framework for presenting templated views of OCDS data. Given a merged OCDS record, OCDS Show can highlight change over time. 
+* [OCDS Show](https://github.com/open-contracting/ocds-show) - a flexible framework for presenting templated views of OCDS data. Given a merged OCDS record, OCDS Show can highlight change over time.
 
 When building an integrated tool that integrates OCDS data into infrastructure project monitoring:
 
 * The [Project Level Data Specification](../project/index.mc) provides a common data structure for recording project-level information;
```

Comment:
Typo: `index.mc`

Reply:
Addressed in https://github.com/open-contracting/infrastructure/pull/46/commits/5c760acb02f31f25ee19ea29362dc314da9f633c